lib/IRGen/GenDecl.cpp

```diff
@@ -1333,9 +1333,8 @@ void IRGenerator::emitLazyDefinitions() {
     assert(LazyFieldDescriptors.empty());
     // LazyFunctionDefinitions are allowed, but they must not be generic
     for (SILFunction *f : LazyFunctionDefinitions) {
-      assert(hasValidSignatureForEmbedded(f));
+      ASSERT(hasValidSignatureForEmbedded(f));
```

Comment:
This is an assert that is active even in production builds, correct? Should I be preferring those in further patches?

Reply:
Right. Yes, I'm always using this `ASSERT` since we have it, because it's really not good to not catch asserts in an ndebug compiler.